docs/victoriametrics/changelog/CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ See also [LTS releases](https://docs.victoriametrics.com/victoriametrics/lts-rel
 
 ## tip
 
+* BUGFIX: `vmselect` in [VictoriaMetrics cluster](https://docs.victoriametrics.com/victoriametrics/cluster-victoriametrics/): fixed not properly ported changes to vmselect API from single-node branch that were introduced in [PR 9267](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/9267).
```

Comment:
It would be better to keep it more user oriented (e.g. what user will see fixed after the update):

```suggestion
* BUGFIX: `vmselect` in [VictoriaMetrics cluster](https://docs.victoriametrics.com/victoriametrics/cluster-victoriametrics/): fix support for `/api/v1/notifiers` endpoint at `vmselect`.
```